pkg/services/alerting/notifier.go

```diff
@@ -22,11 +22,64 @@ var newImageUploaderProvider = func() (imguploader.ImageUploader, error) {
 
 // NotifierPlugin holds meta information about a notifier.
 type NotifierPlugin struct {
-	Type            string          `json:"type"`
-	Name            string          `json:"name"`
-	Description     string          `json:"description"`
-	OptionsTemplate string          `json:"optionsTemplate"`
-	Factory         NotifierFactory `json:"-"`
+	Type            string           `json:"type"`
+	Name            string           `json:"name"`
+	Heading         string           `json:"heading"`
+	Description     string           `json:"description"`
+	Info            string           `json:"info"`
+	OptionsTemplate string           `json:"optionsTemplate"`
+	Factory         NotifierFactory  `json:"-"`
+	Options         []NotifierOption `json:"options"`
+}
+
+// Option holds information about options specific for the NotifierPlugin.
```

Comment:
Should fix linting problem:

```suggestion
// NotifierOption holds information about options specific for the NotifierPlugin.
```